modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/search/TransactionOperationStatus.scala

```diff
@@ -0,0 +1,26 @@
+package org.tessellation.rosetta.server.model.search
+
+import scala.util.Try
+
+import derevo.cats.{eqv, show}
+import derevo.derive
+import enumeratum.{Enum, EnumEntry}
+import io.circe.{Decoder, Encoder}
+
+@derive(eqv, show)
+sealed trait TransactionOperationStatus extends EnumEntry
+
+object TransactionOperationStatus extends Enum[TransactionOperationStatus] {
+  val values = findValues
+
+  case object Accepted extends TransactionOperationStatus
+  case object Reverted extends TransactionOperationStatus
+  case object Canceled extends TransactionOperationStatus
```

Comment:
have you confirmed what about this status?

Reply:
This denotes the status of a transaction operation on the network. Per the [Rosetta docs](https://www.rosetta-api.org/docs/models/Operation.html), a smart contracts may have non-atomic transactions that can be partially applied. This is the status for the whole of the transaction/smart contract, not the individual operations contained within.

Reply:
to keep it in line with its name on Rosetta, it should be `OperationStatus`

Reply:
Renamed.

Reply:
Renamed to `OperationStatus`.